.eslintrc

```diff
@@ -20,6 +20,7 @@
     "import/no-unresolved": 1,
     "no-alert": 1,
     "no-debugger": 1,
+    "react-hooks/exhaustive-deps": 0,
```

Comment:
I kind of liked it. I understand it might happen that you need to go against this rule, but I think I would rather have specific magic comments to disable the rule when needed.
The final decision might also depend on how many situations there are that would need this rule to be disabled, that is something I couldn't check at this time.

Reply:
I found that almost always I have to set the magic comment because I need to remove some direct dependencies to make the component behave correctly. Unfortunately, Volto was architectured in the pre-hooks era and the component props hierarchy are not thought to work with them (yet).

One can always re-add the rule in projects. If you think it's better to leave it we can discuss it or bring it back later!